cmd/tendermint/commands/debug/kill.go

```diff
@@ -121,7 +121,12 @@ func killProc(pid uint64, dir string) error {
 	go func() {
 		// Killing the Tendermint process with the '-ABRT|-6' signal will result in
 		// a goroutine stacktrace.
-		if err := syscall.Kill(int(pid), syscall.SIGABRT); err != nil {
+		p, err := os.FindProcess(os.Getpid())
+		if err != nil {
+			fmt.Printf("failed to find PID to kill Tendermint process: %s", err)
+		}
+		if err = p.Signal(syscall.SIGABRT); err != nil {
```

Comment:
If err is not nil here, p is likely nil

Reply:
Any comment on this?

Reply:
Oops. Looks like we need an `else` branch. Thanks @alessio!

Reply:
https://github.com/tendermint/tendermint/pull/4271